README.md

```diff
@@ -88,15 +92,16 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://stocks-natanim.netlify.app/)
 
```

Comment:
- [ ] **Optional:** You are required to make a video that shows your screen and your face, however, I could not see your face throughout your presentation.
- [ ] **Required** 
- The big letter that blocks your screen [I assume should show your face], covers the webpage such that your presentation is unclear. Kindly re-record this properly.
- Kindly add the video presentation link to your README for proper documentation.

Reply:
Well resolved 👍🏽